Cross-build chill-avro against Scala 2.13 too

hygt opened the pull request:
Hello, currently Scala 2.13 isn't part of the cross Scala versions for the chill-avro module, is there a reason for that?

hygt committed:
Cross-build chill-avro against Scala 2.13 too

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/twitter/chill?pullRequest=704) <br/>All committers have signed the CLA.

hygt committed:
Revert to Scala 2.13.8

hygt commented:
I reverted the Scala version to 2.13.8 in light of https://contributors.scala-lang.org/t/scala-2-13-10-release-planning/5943

And fixed the CI action, I think.

johnynek commented:
Thanks! I approved the CI run. Let's see if it goes green!

johnynek commented:
Thank you very much!

regadas commented:
Can't remember why it hasn't done before ¯\_(ツ)_/¯. Anyway, we should release a maintenance version soon.
